examples/react-todos/client/components/accounts/UserSidebarSection.jsx

```diff
@@ -6,9 +6,11 @@ UserSidebarSection = React.createClass({
       menuOpen: false
     };
   },
+
```

Comment:
Is there a good reason to have all of these blank lines? If you can show me a popular style guide somewhere that suggests having blank lines between methods, I'll go for it.